main.go

```diff
@@ -62,6 +62,8 @@ func validateRequirements(dataplaneBin, haproxyBin string) error {
 func main() {
 	versionFlag := flag.Bool("version", false, "Show version and exit")
 	logLevel := flag.String("log-level", "INFO", "Log level")
+	logRequests := flag.Bool("log-requests", false, "Enable logging requests by Haproxy")
+	logAddress := flag.String("log-address", "", "Syslog address for Haproxy logs (default: log to stderr with this app)")
```

Comment:
Maybe something more descriptive here would be better, such as `log-socket-path`

Reply:
I took the `log-address` term from the HAproxy docs, here https://cbonte.github.io/haproxy-dconv/2.0/configuration.html#log%20(Process%20management%20and%20security , so it seems appropriate to me. Especially as it can be a host[:port], not only a socket path.

So maybe we should prefix `log-address` and also `log-requests` with `haproxy-` to match `haproxy-cfg-base-path` as the parameters affecting the HAproxy itself? It would make the users aware that this is an HAproxy term.

Reply:
Right, it makes sense if the value can also be an address. Staying consistent with haproxy naming is also a good thing.

Reply:
Thanks. But do you agree with adding `haproxy-` prefix to those parameters too?

Reply:
Sorry, I re-read your comment and I think I got it now. 😅 Applied.